### PR TITLE
php: use %E in errorformat instead of postprocess

### PR DESCRIPTION
**Pull request by blueyed**

### CODE REVIEW

**Comment by codeinabox** [comment]
Now that #1006 is merged in and the tests for this refactor are passing, are any other changes required?

File: tests/ft_php.vader [hunk 1]
```diff
@@ -44,5 +47,7 @@ Execute (phpmd):
   AssertEqual len(loclist), 2
   AssertEqual loclist[0].lnum, 16
   AssertEqual loclist[0].text, "Avoid unused parameters such as '$request'."
+  AssertEqual loclist[0].type, 'E'
   AssertEqual loclist[1].lnum, 48
   AssertEqual loclist[1].text, "Avoid variables with short names like $id. Configured minimumlength is 3."
+  AssertEqual loclist[1].type, 'E'
```
Comment (blueyed) [inline tests/ft_php.vader]: I think we should use type=warning for phpmd by default?!
Related: https://github.com/neomake/neomake/issues/584

Comment (codeinabox) [inline tests/ft_php.vader]: I agree as they are code style warnings and not syntax errors.

Comment (codeinabox) [inline tests/ft_php.vader]: Is this something we need to change in this PR or should I create another? Otherwise changes LGTM

Comment (blueyed) [inline tests/ft_php.vader]: I am changing it here, since we have the test here.

**Pull request merged**